File: python/program/Animal.py
# Animal.py
class Animal:
    def __init__(self, species="", name=""):
        self.__species = species
        self.__name = name
    
    def set_species(self, species):
        self.__species = species
    
    def set_name(self, name):
        self.__name = name
    
    def get_species(self):
        return self.__species
    
    def get_name(self):
        return self.__name

File: java/program/Animal.java

public class Animal {
    private String species;
    private String name;
    
    public Animal() {
    }
    
    public Animal(String species, String name) {
        this.species = species;
        this.name = name;
    }
    
    public void setSpecies(String species) {
        this.species = species;
    }
    
    public void setName(String name) {
        this.name = name;
    }
    
    public String getSpecies() {
        return this.species;
    }
    
    public String getName() {
        return this.name;
    }
}